extensions/amp-consent/amp-consent.md

```diff
@@ -261,6 +261,18 @@ AMP will check client cache and server in parallel to find the previous consent
 
 `xssiPrefix`: Causes `<amp-consent>` to strip a prefix from the `checkConsentHref` endpoint's response. If the prefix is not present in the response, then this option will have no effect. `xssiPrefix` can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
 
+#### consentMetadata
```

Comment:
Can you make `consentMetadata` a sub section of this one https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#response

It's an object that's defined within the `checkConsentHref` response. Not something we'd expect from the publishers inline config.